### PR TITLE
remove Mixpanel everywhere

### PR DESCRIPTION
**Pull request by bpostlethwaite**
Removes mixpanel as per internal conversations. @cldougl what are the next steps I need to complete to get this merged?

### CODE REVIEW

**Comment by bpostlethwaite** [comment]
Is this still accurate https://github.com/plotly/documentation/blob/source-design-merge/Contributing.md ?

File: _includes/_new/_page-components/_header-main.html [hunk 1]
```diff
@@ -39,7 +39,7 @@ <h1 class="--name">plotly</h1>
                 <ul>
                     <li><a id="toolsignin" href="/settings" target="_self" class="button --enterprise-quaternary">SIGN IN</a></li>
                     <li><a id="signup" href="/settings" target="_self" class="button --enterprise-quaternary">SIGN UP</a></li>
-                    <li><a href="https://plotly.typeform.com/to/seG7Vb" data-mode="1" target="_blank" onclick="window.mixpanel.track('Typeform demo request');" class="typeform-share button --enterprise-quaternary">REQUEST DEMO</a></li>
+                    <li><a href="https://plotly.typeform.com/to/seG7Vb" data-mode="1" target="_blank" class="typeform-share button --enterprise-quaternary">REQUEST DEMO</a></li>
```
Comment (michaelbabyn) [inline _includes/_new/_page-components/_header-main.html]: This button still works and is formatted properly
sign up buttons

File: _includes/_new/_page-components/_header-main.html [hunk 2]
```diff
@@ -64,7 +64,7 @@ <h1 class="--name">plotly</h1>
                         In</a></li>
                     <li class="tablet"><a id="signup" href="/settings" target="_self" class="button --enterprise-quaternary">SIGN UP</a></li>
                     <li class="tablet"><a href="https://plot.ly/products/cloud/" class="button">UPGRADE</a></li>
-                    <li class="tablet"><a href="https://plotly.typeform.com/to/seG7Vb" data-mode="1" target="_blank" onclick="window.mixpanel.track('Typeform demo request');" class="typeform-share button --enterprise-quaternary">
+                    <li class="tablet"><a href="https://plotly.typeform.com/to/seG7Vb" data-mode="1" target="_blank" class="typeform-share button --enterprise-quaternary">
```
Comment (michaelbabyn) [inline _includes/_new/_page-components/_header-main.html]: I can't get this mobile menu to expand but it doesn't work now anyway. I'll make a separate issue for it.

File: _includes/ad-text.html [hunk 3]
```diff
@@ -23,7 +23,7 @@ <h6 style="text-align: center;">
       <div style="text-align: center;position: relative;">
       		<h1 class="centered" style="margin-bottom: 20px;">plotly.js</h1>
       	<h5>JavaScript library for scientific interactive charts</h5>
-      	<a class="button plotlyjs-download" href="{{site.plotlyjs_download_url}}" onclick='mixpanel.track("PlotlyJS Download",{ "Location": "Banner" });' download>
+      	<a class="button plotlyjs-download" href="{{site.plotlyjs_download_url}}" download>
```
Comment (michaelbabyn) [inline _includes/ad-text.html]: I can't find any references to this code

File: _includes/dashboard-industries-nav.html [hunk 4]
```diff
@@ -31,7 +31,7 @@ <h1 class="dash-title">{{page.industry | capitalize}} Dashboard</h1>
             {% include dashboards-copy/sales-copy.html %}
         </div>
         <div class="col-sm-6" style="text-align:center;">
-          <a href="https://plotly.typeform.com/to/seG7Vb" style="color:{{page.background_color}};border-color:{{page.background_color}} !important;" class="btn-try-it-out typeform-share" data-mode="1" target="_blank" onclick="window.mixpanel.track('Typeform demo request');">Request a demo</a>
+          <a href="https://plotly.typeform.com/to/seG7Vb" style="color:{{page.background_color}};border-color:{{page.background_color}} !important;" class="btn-try-it-out typeform-share" data-mode="1" target="_blank">Request a demo</a>
```
Comment (michaelbabyn) [inline _includes/dashboard-industries-nav.html]: :ok_hand: but I don't think we link to the docs that use this anymore and I guess they're more for SEO.

File: _includes/header-white.html [hunk 5]
```diff
@@ -32,7 +32,7 @@
         </a>
       {% endif %}
 
-      <a href="https://plotly.typeform.com/to/seG7Vb" class="typeform-share button --enterprise-quaternary" data-mode="1" target="_blank" onclick="window.mixpanel.track('Typeform demo request');">REQUEST DEMO</a>
+      <a href="https://plotly.typeform.com/to/seG7Vb" class="typeform-share button --enterprise-quaternary" data-mode="1" target="_blank">REQUEST DEMO</a>
```
Comment (michaelbabyn) [inline _includes/header-white.html]: I don't think this file is used anywhere

File: _posts/plotly_js/2015-07-21-plotlyjs-getting-started.html [hunk 6]
```diff
@@ -27,7 +27,7 @@ <h2>Download</h2>
 
         <p style="text-align: center;">
   			<a class="button plotlyjs-download" href="{{site.plotlyjs_download_url}}"
-  			onclick="onclick=mixpanel.track('PlotlyJS Download',{ 'Location': 'Getting Started Step 1' });" style="display: inline-block;" download>
+  			 style="display: inline-block;" download>
```
Comment (michaelbabyn) [inline _posts/plotly_js/2015-07-21-plotlyjs-getting-started.html]: this button works

**Comment by bpostlethwaite** [comment]
thanks!

**Pull request merged**